README.md

```diff
@@ -536,6 +536,37 @@ Returns an array of arrays, where the *i*th array contains the *i*th element fro
 d3.zip([1, 2], [3, 4]); // returns [[1, 3], [2, 4]]
 ```
 
+### Blur
+
+<a name="blur" href="#blur">#</a> d3.<b>blur</b>() · [Source](https://github.com/d3/d3-array/blob/master/src/blur.js), [Examples](https://observablehq.com/@d3/d3-blur)
+
+Creates a blur transformer, which can blur (or smooth) an *array* of values by repeated application of a moving average transform.
+
+<a name="blur_radius" href="#blur_radius">#</a> *blur*.<b>radius</b>([*radius*])
+
+If *radius* is specified, sets the radius of the transformation: on each iteration, the value of a point is transformed into the mean value of itself and the *radius* points of data surrounding it on the left and on the right (taking into account the edges). If *radius* is not specified, return the current radius. Defaults to 5.
```

Comment:
Any particular reason it defaults to 5?

Reply:
It's the current default in the d3-contour code, but it will change if we switch to the stackblur algorithm (in which radius = 7, iterations=1 gives the same "amount of blurring" as radius=5 with 3 iterations).